solutions/10.js

```diff
@@ -0,0 +1,34 @@
+// Tri Le
+// Find sum of prime numbers 
+
+const isPrime = (number) => {
+  if(number === 2){
```

Comment:
2 is already supported in your for loop, So this if statement would be duplicate code